.github/workflows/ci.yml

```
@@ -29,17 +29,19 @@
-  pages: write
-  security-events: write
+permissions: {}
 
 jobs:
   build:
+    permissions:
+      actions: read
+      checks: write
+      contents: read
+      packages: write
+      #pages: write
```

yamllint (1.35.1)

[warning] missing starting space in comment

(comments)